src/hooks/useAppVersions.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+
+import packageJson from "../../package.json";
+
+const CORE_PACKAGE_URL =
```

Comment:
It should be defined globally in config.ts file or/and in env variable.

Reply:
Actually I think this is a good solution, otherwise we'd have to update it on our side each time. But we don't want to download any information outside of out api, so please retrieve this info from the api

Reply:
I noticed that Dashboard version was already saved in the app, so I used that code. Core version is now taken from api.